Makefile

```diff
@@ -18,5 +18,6 @@ trash-keep: .dapper
 
 .DEFAULT_GOAL := ci
 
-.PHONY: test
+.SECONDEXPANSION:
+.PHONY: $$(TARGETS)
```

Comment:
Forgive me - I haven't worked with Makefiles in over 10 years 😄 - just curious what is the reason for using secondary expansion here as opposed to just `.PHONY: $(TARGETS)`?

Reply:
No reason, it turns out — I was mis-remembering when secondary expansion is necessary! It’s only really useful with automatic variables (or variables which are re-set after the target definition).